src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualNetworkUsageList.md

```diff
@@ -0,0 +1,89 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualNetworkUsageList
+
+## SYNOPSIS
+Gets virtual network current usage.
+
+## SYNTAX
+
+```
+Get-AzureRmVirtualNetworkUsageList -ResourceGroupName <String> -Name <String>
+```
+
+## DESCRIPTION
+The **Get-AzureRmVirtualNetworkUsageList** cmdlet gets per subnet usage for the specified virtual network.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmVirtualNetworkUsageList -ResourceGroupName test -Name usagetest
+
+Get-AzureRmVirtualNetworkUsageList -ResourceGroupName test -Name usagetest
```

Comment:
@EvgenyAgafonchikov can you provide an example of how to use this particular cmdlet when a list if returned
Give a full example something along the line of
$vNetUsageList = Get-AzureRmVirtualNetworkUsageList -ResourceGroupName test -Name usagetest
Because the name has the noun list, but all you are doing is returning 1 PSVirtualNetworkUsage rather the expectation is it should return IEnumerable<PSVirtualNetworkUsage>